### DISCUSSION TITLE
I tried running this project on my Macbook Pro but it got my CPU temps to 80C, would codespaces work better for me?

### DISCUSSION THREAD

**Discussion by hananurrehman** [inventree/InvenTree#8505] [General]
Hi, I'm exploring this project and want to contribute to the tests but I ran into CPU temperature problems. 
Did anyone else run into such issues? Do you recommend codespaces over docker?
(Might sound dumb but I'm a newbie to this)

**Comment by SchrodingersGat** [comment]
@hananurrehman are you trying to build the project using the devcontainer?

Currently there is an issue with one of the required packages (grpc) which basically locks up the CPU trying to install.

### References

- https://github.com/grpc/grpc/issues/34998

**Reply by hananurrehman** [reply]
Yes I built it using the dev container.

**Reply by SchrodingersGat** [reply]
Ok.. We are trying to find a solution to this, might have to build our docker container on a different base image..

You could try installing via the "bare metal" setup guide, rather than under docker?
